docs/ansible/roles/dovecot/getting-started.rst

```diff
@@ -11,55 +11,83 @@ Getting started
    .. contents::
       :local:
 
-Default setup
--------------
-
-If you don't specify any configuration values, the role will setup an IMAP and IMAPS
-service using the certificates provided by `ansible-pki`_. It will further use the
-`ansible-ferm`_ role to open the required network ports with iptables: 143 (IMAP+STARTTLS)
-and 993 (IMAPS). Every user account which is able to login via PAM, can then also
-login via IMAP and access its mails stored as an mbox file in ``/var/mail/<username>``.
-
-LDAP support
-------------
-
-When the :ref:`LDAP environment <debops.ldap>` is configured on a host, the
-``debops.dovecot`` role will automatically switch from system account
-authentication to LDAP-based accounts.
-
 
 Example inventory
 -----------------
 
-You can install Dovecot on a host by adding it to the ``[debops_service_dovecot]`` group
-in your Ansible inventory::
+To enable the :command:`dovecot` service on a host, you need to add it to the
+``[debops_service_dovecot]`` Ansible inventory group:
+
+.. code-block:: none
 
     [debops_service_dovecot]
     hostname
 
+ 
 Example playbook
 ----------------
 
-Here's an example playbook which uses ``debops.dovecot`` role to install Dovecot:
+If you are using the role without DebOps, here's an example Ansible playbook
+that uses the ``debops.dovecot`` role:
 
 .. literalinclude:: ../../../../ansible/playbooks/service/dovecot.yml
    :language: yaml
    :lines: 1,6-
 
+
 Ansible tags
 ------------
 
 You can use Ansible ``--tags`` or ``--skip-tags`` parameters to limit what
-tasks are performed during Ansible run. This can be used after a host was first
-configured to speed up playbook execution, when you are sure that most of the
-configuration is already in the desired state.
+tasks are performed during Ansible runs. This can be used after a host is first
+configured to speed up playbook execution when you are sure that most of the
+configuration has not been changed.
 
 Available role tags:
 
 ``role::dovecot``
   Main role tag, should be used in the playbook to execute all of the role
   tasks as well as role dependencies.
+``role::dovecot:conf``
+  Main configuration tag, should be used in the playbook to execute all of
+  the role tasks relates to configuration creation.
+``role::covecot:conf:sql``
+  `SQL` specific configuration subtag.
+``role::dovecot:conf:ldap``
+  `LDAP` specific configuration subtag.
+``role::dovecot:user``
+  Limited to :command:`dovecot` user configuration tasks.
+``role::dovecot:group``
+  Limited to :command:`dovecot` group configuration tasks.
+
+
+Default setup
+-------------
```

Comment:
I think it would be better to have the "Default setup" and "Other resources" sections between "Example inventory" and "Example playbook", just to prioritise it more.

Reply:
Ok done